deepview_profile/tracking/call_stack.py

```diff
@@ -1,12 +1,19 @@
 import collections
 import inspect
 import os
-
+import re
 import torch
 
 SourceLocation = collections.namedtuple(
     'SourceLocation', ['file_path', 'line_number', 'module_id'])
 
+pattern_list = ["./transformers/models[/\w+/]+\w+.py","./diffusers/models[/\w+/]+\w+.py"]
+
+def find_pattern_match(filename):
```

Comment:
Ideally this should make use of the `any()` function in Python. For example,
```python
return any(re.search(p, filename) for pattern in pattern_list)
```

Reply:
done